#6933

yonatan-dp opened the pull request:
ECMPS UI is not a confidential client, therefore, should not use client tokens and secrets

yonatan-dp committed:
#6933

ECMPS UI is not a confidential client, therefore, should not use client tokens and secrets

yonatan-dp committed:
#6933

Updated so that the sprint deployment version and the main script match.

yonatan-dp committed:
Merge branch 'develop' into fix/#6933-remove-client-secrets

ergjustin reviewed (commented):
Do we need to remove Client Secrets from the database? If so, should we also delete the corresponding client records from the [client-config-data.csv ](https://github.com/US-EPA-CAMD/easey-db-scripts/blob/develop/misc/data-load/client-config-data.csv)file?

yonatan-dp committed:
Removed ecmps-ui and cmapd-ui entries from client-config data load file.

yonatan-dp commented:
@ergjustin good catch Justin 👍
Yes, we are removing the complete entries for ecmps-ui and campd-ui form the database. These two apps are no longer considered clients in the sense that we are using backend APIs as clients. They cannot have client ID/secrets since they will be exposed by definition. 

I have updated the data load csv file to remove both entries.

ergjustin reviewed (dismissed):
Looks good to me!

yonatan-dp committed:
Removed ecmps-ui and cmapd-ui entries from client-config data load file.

yonatan-dp commented:
@ergjustin Please review one more time. I saw that the file did not have an entry for camd-services and I just added it so that we don't forget that during deployment.

yonatan-dp committed:
#6933
Reverted deleting ecmps-ui and campd-ui deletions from client_config because the entry is required for accessing the support emails.

yonatan-dp commented:
Closed because ecmps-ui and campd-ui entries in client_config should remain. They are required for accessing the support emails.
